tests/e2e/java_test.go

```diff
@@ -0,0 +1,128 @@
+package e2e
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"strings"
+)
+
+const files = "examples/java/"
+
+var _ = Describe("odoJavaE2e", func() {
+	const t = "java"
+	var projName = fmt.Sprintf("odo-%s", t)
+	// contains a minimal javaee app
+	const javaeeGitRepo = "https://github.com/lordofthejars/book-insultapp"
+
+	// contains a minimal javalin app
+	const sbGitRepo = "https://github.com/geoand/javalin-helloworld"
```

Comment:
Sneaking in a javelin app, I see! ;)

Reply:
Yeah, I wanted the smallest thing I could find in terms of dependencies :)
Initially I used one of our SB boosters but it took way to long to build on Openshift because Maven was downloading the world...